zone/spell_effects.cpp

```diff
@@ -3791,7 +3797,7 @@ void Mob::DoBuffTic(const Buffs_Struct &buff, int slot, Mob *caster)
 
 	const SPDat_Spell_Struct &spell = spells[buff.spellid];
 
-	std::string export_string = fmt::format(
+	const auto& export_string = fmt::format(
```

Comment:
I know this is used in multiple so we defined it out of band, but most people don't even use these subroutines so this is largely wasted every time. Making note to move these inline and duplicate them for each